target_chains/ton/contracts/tests/utils/pyth.ts

```diff
@@ -36,3 +36,26 @@ export const HERMES_BTC_ETH_UPDATE =
 
 export const BTC_PRICE_FEED_ID =
   "0xe62df6c8b4a85fe1a67db44dc12de5db330f7ac66b72dc658afedf0f4a415b43";
+
+export const TEST_GUARDIAN_ADDRESS1 =
+  "0x686b9ea8e3237110eaaba1f1b7467559a3273819";
+
+// A Pyth governance instruction to authorize governance data source transfer signed by the test guardian #1.
```

Comment:
not for this PR but for future PRs: this copying is brittle. We should at some point (during the development of TON) elevate the rust test generating code from starknet and change it to generate this file.